packages/ui-kit-theme/src/overrides/MuiButtonGroup.ts

```diff
@@ -0,0 +1,60 @@
+import { alpha, useTheme } from '@mui/material';
+
+declare module '@mui/material/ButtonGroup' {
+  interface ButtonGroupPropsVariantOverrides {
+    control: true;
+  }
+}
+
+export default {
+  defaultProps: {
+    disableFocusRipple: true,
+    disableRipple: true,
+  },
+  variants: [
+    {
+      props: { variant: 'primary' as const },
+      style: () => {
+        const theme = useTheme();
```

Comment:
I'm not sure the hook should be used outside a react component. This function should accept `{ theme, ownerState}` as an argument. It's probably safer to get the theme from there.